R/internals.R

```diff
@@ -280,7 +281,7 @@
     clean_json <- clean_json[grep(script_name, clean_json)] %>% stringi::stri_unescape_unicode()
     clean_json <- qdapRegex::rm_square(clean_json, extract = TRUE, include.markers = TRUE) %>% unlist() %>% stringr::str_subset("\\[\\]", negate = TRUE)
 
-    out_df <- lapply(clean_json, .fromJSON) %>% do.call("rbind", .)
+    out_df <- lapply(clean_json, jsonlite::fromJSON) %>% do.call("rbind", .)
```

Comment:
the one place i found where we were using `.fromJSON()` outside of Fotmob functions. `fromJSON()` shouldn't have any issues here since it's not being used on raw URLs